src/main/java/edu/hm/hafner/analysis/LookaheadParser.java

```diff
@@ -113,4 +115,25 @@ protected boolean isLineInteresting(final String line) {
     protected Report postProcess(final Report report) {
         return report;
     }
+    
+    /**
+     * MSYS make on windows replaces "c:/" by "/c/" - here we change it back
+     * 
+     * @param path A path to be fixed
+     * @return The fixed path
+     */
+    private String fixMsysTypePath(String path) {
+        if (isWindows() && path.matches("/[A-Za-z]/.*")) {
```

Comment:
This is not that easy. IsWindows checks if the Jenkins process that parses the file is on Windows. But Jenkins can run on Unix while the agent runs on Windows.

Reply:
Right, sorry. This is what I get for commenting before I've had any sort of coffee :)

Reply:
@uhafner Is this a general comment about the code, or about using commons library? I guess the point is that this code must run on the machine where the workspace is located.

Reply:
This is a general problem with your algorithm. You need to detect from the log itself if the log output is from a Windows system. The operating system that is executing the log parser can't help here as it might be a different one.

E.g., if someone has Jenkins running on Windows, then every log produced by a Linux agent will get the paths replaced.

Reply:
The whole point here is to generate correct links to the file. If the log is from a different system, then this can never work anyway, so it won't matter.

Reply:
No, this is not correct. Currently a Linux log will be parsed correctly even if Jenkins is running on Windows. If your patch is applied then this is not true anymore.

Reply:
Any ideas on how this could be fixed then?

Reply:
Probably better to just run the log file through a sed script before processing than having the plugin support weird behaviour. I'll close the pull request.